client/extensions/woocommerce/woocommerce-services/state/shipping-label/selectors.js

```diff
@@ -260,11 +303,99 @@ const getPackagesErrors = values =>
 		return errors;
 	} );
 
-export const getCustomsErrors = customs => {
-	if ( ! customs.dummyFieldChecked && customs.submitted ) {
-		return { dummyField: 'This must be selected to continue' };
-	}
-	return {};
+export const getCustomsErrors = (
+	packages,
+	customs,
+	destinationCountryCode,
+	destinationCountryName
+) => {
+	const usedProductIds = uniq(
+		flatten( map( packages, pckg => map( pckg.items, 'product_id' ) ) )
```

Comment:
We need to account for `variations_id` here, as described in:

https://github.com/Automattic/woocommerce-services/projects/2#card-11490865 

https://github.com/Automattic/wp-calypso/pull/24974#discussion_r203568135

https://github.com/Automattic/wp-calypso/pull/24974#discussion_r203568394

But I'm happy to merge this and fix in a later PR if that works better for you.